src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -22,60 +20,47 @@ const OverviewRoute: SFC<OverviewRouteProps> = props => {
   const { artist } = props
 
   return (
-    <Subscribe to={[FilterState]}>
```

Comment:
I guess right now this doesn't need to listen to the current filter state (since it starts out with some defaults).

I think if we want to hook up URL parsing we'll need to then pass in some specific filter state (and can remove the nullable constraint added in https://github.com/artsy/reaction/pull/944/files#diff-e3efa0e5ac989dfe1a9073a3fc6c8384R18) , but thats a follow up. Either way I don't think we'd want to listen to the filter state here.

You can also go the route of 'url updates trigger a state change, and things subscribed to the state change (like the filter) can then do whatever'. In that case, maybe we'll bring back some sort of state listener here?